### PR TITLE
Source

### PR DESCRIPTION
**Pull request by rogeralmeida**

### CODE REVIEW

**Comment by rogeralmeida** [comment]
Hey @das2011 can you review this blogpost in your free time for me? I wanna make sure that I not making stupid basic English mistakes

File: source/_posts/2015-06-15-my-e-mail-policy.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Keep email closed
+I don't keep my email inbox open during the working hours. I open it just when I have to. I also disabled the email notification in my cellphone. That prevents me from check email whatever I see that a new email arrived.
+
+##Schedule time to check email
+I check my email twice a day, at 10:00 am and 04:00 pm. I have reminders in my calendar . That helps me organize my day and prevents me from being fooled by my weak memory.
+
+##Time box email time
+As an adopter of the [Pomodoro Technique](http://pomodorotechnique.com/) I keep track of the time I spend with email. What I noticed is that I need almost two pomodoros  to read/respond/forward/archive my emails. That means that I need almost two hours during my day only to deal with my email inbox.  That was so true when I was working for [ThoughtWorks](http://www.thoughtworks.com/). Now that I'm working for [DiUS](http://dius.com.au/) I'm trying not to sign in to so much email's lists. I aim to keep my email time in one pomodoro.
+
```
Comment (das2011) [inline source/_posts/2015-06-15-my-e-mail-policy.markdown]: I'm trying not to sign up to so many email lists

Comment (das2011) [inline source/_posts/2015-06-15-my-e-mail-policy.markdown]: I am to keep my email time to one pomodoro

File: source/_posts/2015-06-15-my-e-mail-policy.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+##So far, so good
+Till here I described what I have been doing for one year. It's working fine and I notice that my productive increased since then. Before I was checking email every time I alternated windows. Now, I can alternate windows without the temptation of checking email. But there is one last step that  I wanna try...
```
Comment (das2011) [inline source/_posts/2015-06-15-my-e-mail-policy.markdown]: Up until here I've described..... It's working fine and I've noticed that...

File: source/_posts/2015-06-15-my-e-mail-policy.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+##Vacation Responder On
+From today on I'll keep my Vacation Responder On in my professional email. This is the message that I will send to who send me email:
```
Comment (das2011) [inline source/_posts/2015-06-15-my-e-mail-policy.markdown]: This is the message that I will send to those who send me email:

File: source/_posts/2015-06-15-my-e-mail-policy.markdown (hunk 1 above)
Comment (rogeralmeida) [inline source/_posts/2015-06-15-my-e-mail-policy.markdown]: In the second, what I wanna say actually is: "My aim is to keep my email time to one pomodoro"
Does that sound better?

**Pull request merged**